docs/content/using-npm/scope.md

```diff
@@ -79,9 +79,22 @@ If you wish, you may associate a scope with a registry; see below.
 
 #### Publishing public scoped packages to the primary npm registry
 
-To publish a public scoped package, you must specify `--access public` with
-the initial publication. This will publish the package and set access
-to `public` as if you had run `npm access public` after publishing.
+Publishing to a scope, you have two options:
+
+- Publishing to your user scope (example: `@username/module`)
+- Publishing to an organization scope (example: `@org/module`)
+
+If publishing a public module to an organization scope, you must
+first either create an organization with the name of the scope
+that you'd like to publish to or be added to an existing organization
+with the appropriate permisssions. For example, if you'd like to 
+publish to `@org`, you would  need to create the `org` organization 
+on npmjs.com prior to trying to publish.
+
+Once you will need to 
+specify `--access public` with the initial `npm publish` command.
+This will publish the package and set access to `public` as if 
+you had run `npm access public` after publishing.
```

Comment:
```suggestion
Scoped packages are not public by default.  You will need to specify
`--access public` with the initial `npm publish` command.  This will publish
the package and set access to `public` as if you had run `npm access public`
after publishing.  You do not need to do this when publishing new versions of
an existing scoped package.
```